### ISSUE TITLE
Add accessibility support

### ISSUE THREAD

**Issue by wiredwiz** [wiredwiz/FastColoredTextBox#3]
Add support for screen reader access to the Fast Colored TextBox.

**Comment by wiredwiz** [comment]
Unfortunately this control currently lacks any support for accessibility via screen readers.
